src/main/java/game/domain/Board.java

```diff
@@ -0,0 +1,48 @@
+package game.domain;
+
+import java.util.Arrays;
+
+public class Board {
+    private static final String INVALID_INPUT = "입력이 잘못되었습니다, 명령(h, j, k, l, q)";
+    private final Positions positions;
+    private final OrderChecker orderChecker;
+    private Pointer pointer;
+    private Moves moves;
+
+    public Board(Numbers numbers) {
+        this.positions = new Positions(numbers);
+        this.orderChecker = new OrderChecker();
+        this.pointer = Pointer.from(positions);
+        this.moves = new Moves();
+    }
+
+    /**
+     * 입력한 방향으로 포인터를 이동한다.
+     * @param input 입력방향
+     */
+    public void movePointer(String input) {
+        Directions directions = Directions.from(input);
```

Comment:
```suggestion
    public void movePointer(Directions directions) {
```

input을 바로 받는 것보다는, Directions 객체를 받는게 가독성 및 input에 대해서 정확한 값을 받을 수 있어보이는데 어떠신가요?

Reply:
좋은 의견인 것 같아요. Directions 객체를 활용하도록 수정했습니다.  [1e2caab](https://github.com/etff/fifteen-puzzle/pull/1/commits/1e2caabc123a50b17c749a234042318885ba41f2)